src/components/Cart/Cart.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { CartContext } from './CartContext';
+import './Cart.scss';
+import { Link } from 'react-router-dom';
+
+interface Props {
+  goods: Good[];
+}
+
+export const Cart: React.FC<Props> = ({ goods }) => {
+  const {
+    cart, changeGoodCount, selectGood, clearCart,
+  } = useContext(CartContext);
+  const [goodForBuy, setGoodForBuy] = useState<Good[]>(goods
+    .filter(good => cart
+      .some(prod => prod.id === good.id)));
```

Comment:
Code style порада.
Скільки дужок в одному рядку відкрилося стільки і має закритися в одному рядку, приклад:
```suggestion
  const [goodForBuy, setGoodForBuy] = useState<Good[]>(goods
    .filter(
      good => cart.some(prod => prod.id === good.id),
    ),
  );
```
```suggestion
  const [goodForBuy, setGoodForBuy] = useState<Good[]>(goods
    .filter(good => cart.some(prod => prod.id === good.id)),
  );
```

Reply:
+